test/model/src/include/model/kv_table.h

```diff
@@ -114,12 +128,13 @@ class kv_table {
     int remove(kv_transaction_ptr txn, const data_value &key);
 
     /*
-     * kv_table::fix_commit_timestamp --
-     *     Fix the commit timestamp for the corresponding update. We need to do this, because
-     *     WiredTiger transaction API specifies the commit timestamp after performing the
+     * kv_table::fix_timestamps --
+     *     Fix the commit and durable timestamps for the corresponding update. We need to do this,
+     *     because WiredTiger transaction API specifies the commit timestamp after performing the
      *     operations, not before.
      */
-    void fix_commit_timestamp(const data_value &key, txn_id_t txn_id, timestamp_t timestamp);
+    void fix_timestamps(
```

Comment:
This API is a little confusing - we now take a timestamp and a durable timestamp. It's possible to puzzle out that `timestamp` must be the commit timestamp since the comments says it's commit and durable, and the other is `durable_timestamp`. But that's a lot of work, and easy to get wrong at the callsite.

I would suggest either:
- Having different `fix_commit_timestamp` and `fix_durable_timestamp` functions
- A `fix_timestamp` function that takes a `timestamp_t` and an enum describing which timestamp to set

Reply:
OK, that makes more sense now. Maybe just rename the `timestamp` parameter to `commit_timestamp` then?

Reply:
Ah, that's right. I thought I renamed it, but I renamed it in only one place and not here. Ok, done. Thanks for catching that!